scalableminds/webknossos
# Restrict user visibility (non-teammanagers see only themselves)

### URL of deployed dev instance (used for testing):
- https://restrictuservisibility.webknossos.xyz

### Steps to test:
- create multiple accounts, create exploratives / tasks (with scripts)
- users without teammanager/admin rights should still be able to perform all their tasks
- still when they hit /api/users the list should contain only themselves

### Issues:
- fixes #3374 

------
